### ISSUE TITLE
DNA volume: ROI for Otsu thresholding

### ISSUE THREAD

**Issue by TobiasKletter** [tischi/spindle3d#17]
Quick and dirty testing of decreasing ROIs for Otsu thresholding:

Sample 1 (High Signal to noise ratio, other cells close by)
![Screenshot 2021-02-18 at 16 24 06](https://user-images.githubusercontent.com/45814595/108379478-46e95880-7206-11eb-96cb-06e5bbb421d8.png)

Sample 2 (High Signal to noise ratio, no other cells)
![image](https://user-images.githubusercontent.com/45814595/108380738-6b920000-7207-11eb-954f-3a71e4a735bc.png)

Sample 3 (Moderate Signal to noise, other cells present)
![image](https://user-images.githubusercontent.com/45814595/108381658-4ce03900-7208-11eb-8fb4-1a9d7d43cd18.png)

Seems fairly robust, but there are some changes in area size and they might be quite dramatic when applied to the whole volume?
So my feeling is, it is indeed good practice to restrict the ROI such that outside signals don't interfere.

I will do this a bit more systematic, and with a nice plot that we can add to the paper...

**Comment by tischi** [comment]
Yeah, maybe in 3D for volume measurements the relative effect could be larger, hard to say though...

Maybe you'd have to look for examples, and I would need to add an option to the advanced code to compute the threshold in the whole image. Then we could compare the effects in 3D.
And, I am not sure whether it's worth it. I am pretty convinced what we are doing is best practice even if the difference in our data may be small.
